background.js

```diff
@@ -27,7 +27,7 @@ chrome.runtime.onMessage.addListener(function (message) {
 chrome.omnibox.onInputChanged.addListener(omni.suggest.bind(omni));
 
 chrome.omnibox.onInputStarted.addListener(function () {
-    if (_.isUndefined(localStorage.setup)) {
+    if (_.isUndefined(localStorage.getItem('setup'))) {
```

Comment:
We were both wrong. For some reason i wasn't setting the `setup` flag after you made a decision.